### PR TITLE
added name/link of channel/room in email notifications

### PR DESCRIPTION
**Pull request by danielrohers**
@RocketChat/core 

Closes [E-Mail notifications: include name/link of channel/room #3704](https://github.com/RocketChat/Rocket.Chat/issues/3704)

Hello, add link to the channel that was mentioned...

screen shot 2016-07-19 at 13 48 31

### CODE REVIEW

**Comment by sampaiodiego** [comment]
thanks for the contribution @danielrohers 

I don't like the idea of repating the subject on email body, specially because it adds the "server name" (which is already in the body header)..

I think we should change the translations for `Offline_Mention_Email` and `Offline_DM_Email` to not include the `[__site__]`.. So we can use it in both places (subject and body) but adding the "server name" where it's needed. What you think?

**Comment by danielrohers** [comment]
I think it's a good idea @sampaiodiego 

Is there any way to make this change with i18n files or I need to do manually?

I saw not treated the DM situation, I will adjust it too.

**Comment by sampaiodiego** [comment]
> Is there any way to make this change with i18n files or I need to do manually?

@danielrohers have to do manually :disappointed: but a replace in all files should do the trick :smiley:

**Comment by danielrohers** [comment]
@sampaiodiego preview messages. `Site_Name` believe to be interesting in the subject to easily see the contexts...

screen shot 2016-07-20 at 20 20 54

screen shot 2016-07-20 at 20 21 10

**Comment by sampaiodiego** [comment]
cool @danielrohers 

can you please rebase and fix the conflicts?

**Comment by rodrigok** [comment]
Hi @danielrohers, what do you think about adding a center (big and green?) button bellow all text with text "Go to channel" instead of that small link?

Another thing, can you remove that `>` on the top of the email?

**Comment by danielrohers** [comment]
Hi @rodrigok,

Great! I better keep the "RC blue" color and with "Go to message" since it can use in both cases (DM / Channel).

What do you think?

**Comment by rodrigok** [comment]
@danielrohers Ok, better, go ahead 😄

**Comment by danielrohers** [comment]
Hey guys

I made the changes, I believe that really got much better :)

@sampaiodiego @rodrigok 

screen shot 2016-07-25 at 11 47 34
screen shot 2016-07-25 at 11 47 43

File: packages/rocketchat-lib/server/lib/sendEmailOnMessage.js [hunk 1]
```diff
@@ -22,14 +22,30 @@ RocketChat.callbacks.add('afterSaveMessage', function(message, room) {
...
+			'padding: .5em;',
+			'background-color: #04436a;',
+			'display: block;',
+			'width: 10em;',
+			'text-align: center;',
+			'text-decoration: none;',
+			'margin: auto;',
+			'margin-bottom: 8px;'
+		].join(' ');
+		return `<a style="${ style }" href="${ process.env.ROOT_URL }${ path }">GO TO MESSAGE</a>`;
```
Comment (rodrigok) [inline packages/rocketchat-lib/server/lib/sendEmailOnMessage.js]: @danielrohers can you change this to use TAPi18n to `GO TO MESSAGE` and add this text to en.i18n.json?

**Comment by rodrigok** [comment]
Can you solve the conflicts too?

**Comment by rodrigok** [comment]
@danielrohers Awesome man :)

**Pull request merged**